gateway/ln-gateway/src/lib.rs

```diff
@@ -993,7 +985,8 @@ impl Gateway {
             client
                 .get_first_module::<GatewayClientModule>()
                 .register_with_federation(
-                    route_hints,
+                    // Route hints will be updated in the background
+                    Vec::new(),
```

Comment:
Good enough, but if this overwrites the existing registration, would be better to not overwrite with empty value, even if just for a moment, as it might lead to flakiness (mostly worried about prod).

Reply:
I think that would be difficult to do, since we check if the federation is already connected or not in the code above.

Reply:
Oh, OK. Nvmd then.